spec/double_entry/reporting/aggregate_array_spec.rb

```diff
@@ -40,19 +40,34 @@ module Reporting
             describe 'reuse of aggregates' do
               let(:years) { TimeRangeArray.make(range_type, start, finish) }
 
-              context 'and some aggregates were created previously' do
```

Comment:
I think this context line is worth keeping. It explains the reason for the setup code.